tuna/miopen_tables.py

```diff
@@ -488,9 +482,8 @@ class BNGolden(BASE, GoldenMixin):
   """Golden table for batch norm"""
   __tablename__ = "bn_golden"
   __table_args__ = (UniqueConstraint("golden_miopen_v",
-                                     "find_db",
                                      "config",
-                                     "session",
```

Comment:
session shouldn't be part of the golden table's unique constraints. There can be multiple session with overlapping information within arch/num_cu/config/solver, so it would confound the table.